docs-kits/kits/Modular Production Kit/page_adoption-view.md

```diff
@@ -15,74 +15,164 @@ sidebar_position: 2
+In the case of the overarching example, the *ProvideProductionForecastData* model provides new information via notifications, one of the three specified communicationModes, indicating that there has been a change in the estimated production date. The *positionId* specifies that the box is impacted. Along with the reason for the delay, a new estimated date is provided enabling the avoidance of useless commitment of production capacities and the optimization of production processes through more precise planning.
+
+### Production Tracking
+
+The Production Tracking aspect of the Shop Floor Information Service features again one data exchange mechanism, which in turn consists of two semantic models, namely the *GetProductionTrackingData* model and the *ProvideProductionTrackingData* model.
+
+All partners need the same semantic models for a common understanding of the information exchange. Customers or third parties specify within the *GetProductionTrackingData* model the required information about a manufacturing step. This requires that both Modular Production as well as the customer have a common understanding about the product to be manufactured. This common knowledge is referenced for the request.  Modular Production interprets this request and replies with the *ProvideProductionTrackingData* model. Depending on the customer’s request, Modular Production sends the requested information.
+
+The *GetProductionTrackingData* is sent from a partner to Modular Production with a HTTP request to receive production tracking information. The *ProvideProductionTrackingData* model is containing the relevant information like *processParameterName*, *processParameterValue* and *processParameterQuality*
+
+The semantic model can be found here:
+
+- [io.catenax.shopfloor_information.get_production_tracking](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shopfloor_information.get_production_tracking)
+- [io.catenax.shopfloor_information.provide_production_tracking](https://github.com/eclipse-tractusx/sldt-semantic-models/tree/main/io.catenax.shopfloor_information.provide_production_tracking)
+
+The *GetProductionTrackingData* model must be implemented by all participants who wish to use the production tracing aspect of the Shop-Floor-Information-Service as a Modular Production, a customer or a participating third party. Companies, which use the Shop-Floor-Information-Service as a customer or third party must be able to send *GetProductionTrackingData* request, whereas partners using the Shop-Floor-Information-Service as a Modular Production must be able to receive the *GetProductionTrackingData* request.
+
+![SIS.Tracking-Parameter](res/SIS.Tracking-Parameter.svg)
```

Comment:
This image is not displayed in the rich cliff - please check for all pictures below, as many of them don't seem to be rendered from the link you provided.

Reply:
images appear to render with docusaurus - can you elaborate in which case the images do not render correctly?

![Bildschirmfoto 2024-05-23 um 14 09 53](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/assets/1258026/449ea050-72ad-4c97-9f7d-c1636905e682)

Reply:
In this case, it works fine! Thanks for building it to verify!